src/clims/api/endpoints/substance.py

```diff
@@ -27,24 +30,19 @@ def handle_results(qs):
             for entry in qs:
                 wrapper = self.app.substances.to_wrapper(entry)
                 json = SubstanceSerializer(wrapper).data
-                json['position'] = {'index': "A:1", 'container': {'name': "cont1"}}
-                if 'volume' not in json['properties']:
-                    json['properties']['volume'] = random.randint(0, 500)
-                if 'sample_type' not in json['properties']:
-                    json['properties']['sample_type'] = random.choice(["Amplicon", "WGS", "WES"])
-                if 'priority' not in json:
-                    json['priority'] = random.randint(1, 10)
-                if 'days_waiting' not in json:
-                    json['days_waiting'] = random.randint(0, 100)   # NOTE: We would have created and origin_created instead
-                if 'container' not in json['properties']:
-                    json['properties']['container'] = random.choice(["container 1", "container 2", "container 3"])
+                json['position'] = {'index': "A:1",
```

Comment:
Don't we have everything we need to get rid of this now?

Reply:
Yeah, I'll remove the rest.